frontend/client/src/components/NavBar.js

```diff
@@ -77,23 +77,28 @@ const NavBarBase = observer((props) => {
         open={open}
         onClose={handleClose}
       >
-        <MenuItem className="menu-link" style={{ marginTop: 22 }} onClick={() => onClickMenuItem(0)}>
+        <MenuItem
+          id="diagnosis-verification-codes"
+          className="menu-link"
+          style={{ marginTop: 22 }}
```

Comment:
move style to scss

Reply:
Since it's not a change I made in this PR I'm going to leave this for now and hold off until somebody picks [this issue](https://github.com/covidwatchorg/portal/issues/449) up